docker/vttestserver/Dockerfile.mysql80

```diff
@@ -55,4 +55,7 @@ COPY --from=builder --chown=vitess:vitess /vt/install /vt
 VOLUME /vt/vtdataroot
 USER vitess
 
-CMD /vt/bin/vttestserver -port $PORT -keyspaces $KEYSPACES -num_shards $NUM_SHARDS -mysql_bind_host ${MYSQL_BIND_HOST:-127.0.0.1} -mysql_server_version ${MYSQL_SERVER_VERSION:-8.0.21-vitess}
+COPY docker/vttestserver/setupVschemaFolder.sh /vt/setupVschemaFolder.sh
+COPY docker/vttestserver/runMysql80.sh /vt/run.sh
```

Comment:
You could combine the two run scripts into one that takes an arg and use it as `mysql_server_version`.

Reply:
Agreed

Reply:
done